### ISSUE TITLE
Tunnelblick does not start automatically

### ISSUE THREAD

**Issue by eric-nieuwland** [Tunnelblick/Tunnelblick#796]
**Describe the bug**
Tunnelblick does not start automatically at boot/login after update to MacOS Sonoma 14.4.1.

**To Reproduce**
1. Start Mac and login
2. Start Tunnelblick - optionally: connect to any configured VPN
3. Restart Mac and login

**Expected behavior**
Tunnelblick should start automatically

**Desktop (please complete the following information):**
 - MacOS: Sonoma 14.4.1
 - Tunnelblick: 4.0.1 (build 5971)

**Comment by jkbullard** [comment]
@eric-nieuwland wrote:
> See error

What error?

**Comment by jkbullard** [comment]
@eric-nieuwland - I cannot reproduce this on a new installation of 14.4.1 and a new installation of Tunnelblick 4.0.1.

**Comment by eric-nieuwland** [comment]
Hmmm now that I see your name I wonder:
Could it be that I need to enable/allow "Jonathan Bullard" to run in the background?

**Comment by jkbullard** [comment]
Yes, that's exactly it. macOS started listing programs which had not been highly visible to the user, and lets the user disable them. Because of other macOS changes, it lists one of Tunnelblick's two background programs with my name because I digitally sign the program (see Issue #771). We will be releasing a Tunnelblick beta soon that fixes this problem.

For anyone else who would like to disable one or both of Tunnelblick's background programs, please see [Tunnelblick's Background Programs](https://tunnelblick.net/cBackgroundPrograms.html).
